MapboxCoreNavigation/RouteController.swift

```diff
@@ -854,13 +854,13 @@ extension RouteController: CLLocationManagerDelegate {
 
         routeProgress.currentLegProgress.currentStepProgress.userDistanceToManeuverLocation = userAbsoluteDistance
 
-        guard let spokenInstructions = routeProgress.currentLegProgress.currentStep.instructionsSpokenAlongStep else {
+        guard let spokenInstructions = routeProgress.currentLegProgress.currentStepProgress.remainingSpokenInstructions else {
             print("The directions request was made without `includesVoiceInstructions` enabled. This will prevent users from getting voice instructions. It's recommended to make your directions request via `NavigationRouteOptions()`.")
```

Comment:
@bsudekum, this log statement is a bit misleading: in the scenario that previously crashed, `includesVoiceInstructions` was set, so this is just unnecessary spew.

Reply:
This is true.